build.zig

```diff
@@ -521,6 +538,41 @@ fn build_tigerbeetle_executable_multiversion(b: *std.Build, options: struct {
     return build_multiversion.addPrefixedOutputFileArg("--output=", basename);
 }
 
+// Downloads a pre-build llvm-objcopy from <https://github.com/tigerbeetle/dependencies>.
+fn build_tigerbeetle_executable_get_objcopy(b: *std.Build) std.Build.LazyPath {
+    const llvm_objcopy_artifact: struct { name: []const u8, checksum: u256 } =
```

Comment:
Hm - x86_64 macos and aarch64 Linux?

Reply:
(to clarify, won't be a huge problem to add those, but I'd rather do that lazily, once we need. There's a chance that we'll just rewrite-it-in-zig faster)